lib/Cargo.toml

```diff
@@ -19,7 +19,7 @@ log = "0.4.17"
 tokio = { version = "1.25.0", optional = true }
 bincode = { version = "2.0.0-rc.3", optional = true }
 async-stream = "0.3.5"
-
+dashmap = "*"
```

Comment:
Could you use exact version of dashmap's crate?

Reply:
wait leme fix this

Reply:
it will automatically use the latest one